doc/source/spectra.rst

```diff
@@ -117,12 +137,12 @@ a specific spectrum as a ``Spectrum1D``, or a subset of spectra as
 another ``Spectrum1DCollection``, for example, to plot only specific
 spectra:
 
-.. code-block:: py
+.. testcode:: coh_incoh_pdos
 
   from euphonic import Spectrum1DCollection
   from euphonic.plot import plot_1d
 
-  spec1d_col = Spectrum1DCollection.from_json_file('dos.json')
+  spec1d_col = Spectrum1DCollection.from_json_file('coherent_pdos.json')
```

Comment:
Am not sure if I'm just being incompetant, but I can't find `coherent_pdos.json`, (though I can find `dos.json`). Is it me, or does it indeed need renaming?